### DISCUSSION TITLE
What the benifit from changing "TEXT" to "TEXTVIEW"?

### DISCUSSION THREAD

**Discussion by ameaninglessname** [Sixze/ALS-Refactored#252] [Q&A] [answered]
I notice there are changes (686d84efa4164d0105d99333e3bdb9e8e1aa1442) about this, and want to learn more about it.

**Comment by Sixze** [comment, marked as answer]
`TEXTVIEW()` calculates the length of a string from a string literal at compile time, so constructing `FName` and `FString` from it should be slightly faster because there is no need to calculate the length of the string at runtime.

**Reply by ameaninglessname** [reply]
Got it, thanks!
